nvidia/nvidia-graphics-setup

```diff
@@ -88,9 +88,29 @@ nvidia_blacklist_check_by_pci_id() {
   return 0
 }
 
+# Return 1 if module name passed as argument is blacklisted through the kernel
+# command line with modprobe.blacklist=module
```

Comment:
Please note here that you also detect if its blacklisted via the modprobe.blacklist=a,b,c notation. Otherwise at initial glance this seems like a painful way just to do the simple string match that is mentioned.